src/server/routes/queueDelete.ts

```diff
@@ -121,12 +124,25 @@ export async function enqueueSavedItemIds(
 
   // If there's any remaining, send to SQS
   if (sqsEntries.length) {
-    sqsSends.push(sqsSendBatch(sqsEntries));
+    sqsCommands.push(buildSqsCommand(sqsEntries));
   }
 
   try {
-    await Promise.all(sqsSends);
+    await Promise.allSettled(
+      sqsCommands.map((command) => {
+        // Handle logging individual errors as the promises are resolved
+        return sqs.send(command).catch((err) => {
+          const message = `QueueDelete: Error - Failed to enqueue saved items for userId: ${userId} (command=\n${JSON.stringify(
+            command
+          )})`;
+          Sentry.addBreadcrumb({ message });
+          Sentry.captureException(err);
+          console.log(message);
+        });
+      })
+    );
   } catch (e) {
+    // Also provide top-level logging
```

Comment:
this now doesnt need a try-catch right as each promise would handle their own error?